product_docs/docs/biganimal/release/getting_started/overview.mdx

```diff
@@ -0,0 +1,116 @@
+
+## Option 1 - Consolidated instructions
+
+Use the following high-level steps to connect BigAnimal to your cloud account and begin using your new database. The steps vary depending on whether you are using your own cloud account or the BigAnimal portal.
+
+1. *If using your own cloud account:* Ensure you have the correct access permissions for your cloud account. 
+    - Azure&mdash;Global Administrator or Privileged Role Administrator and you must be a Subscription Owner
+
+    - AWS&mdash;IAMFullAccess policy and ServiceQuotasFullAccess policy
+
+    - Google Cloud&mdash;roles/owner or at least the following combined roles:
+
+        - roles/iam.serviceAccountCreator
+
+        - roles/iam.serviceAccountKeyAdmin
+
+        - roles/iam.roleAdmin
+
+        - roles/resourcemanager.projectIamAdmin
+        
+        - roles/compute.viewer
+
+1. Create an EDB account. For more information see [Create an EDB account](https://www.enterprisedb.com/docs/biganimal/latest/free_trial/detail/create_an_account/).
+
+1. After signing into your EDB account for the first time, take one of the following steps:
+    - Continue to use BigAnimal as your portal and skip to the next step.
+
+    - Set up your own identity provider. For more information see [Setting up your identity provider](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/identity_provider/).
+    
+1. Set up a CLI. See [Using the BigAnimal CLI](https://www.enterprisedb.com/docs/biganimal/latest/reference/cli/). 
+   Although we recommend using a CLI, it is not essential.
+
+1. *If using your own cloud account:* Prepare your cloud to work with BigAnimal. You must do this every time you create a new cluster. The required steps vary by cloud provider:
+    - Azure&mdash;see [Preparing your Azure account](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/preparing_cloud_account/01_preparing_azure/).
+
+    - AWS&mdash;see [Preparing your AWS account](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/preparing_cloud_account/02_preparing_aws/).
+
+    - Google Cloud&mdash;see [Preparing your Google Cloud account](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/preparing_cloud_account/preparing_gcp/).
+
+1. *If using your own cloud account:* Connect to your cloud. See [Connecting to your cloud](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/02_connecting_to_your_cloud/).
+
+1. *If using your own cloud account:* Activate and manage regions. See [Managing regions](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/activating_regions/). 
+
+1. Create a cluster. See [Creating a cluster](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/creating_a_cluster/).
+
+1. Use your cluster. See [Using your cluster](https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/)
+
+## Option 2 - Two procedures based on cloud approach
+
+### Getting started using the Big Animal portal 
```

Comment:
maybe "Getting started running in BigAnimal's cloud account"

Reply:
I didn't like the "running" word so I inserted "using". But maybe you have other ideas?